### DISCUSSION TITLE
Someting missing on fresh install of HA

### DISCUSSION THREAD

**Discussion by phateks** [punxaphil/custom-sonos-card#248] [Q&A]
Hello to all, and hello johanfrick,
after an update of HA and previously installed v6.13.x, this card gave me white nights :), so i started a fresh rebuild of everything in HA but for the love of god can't seem to make this card work properly. 
Installed / reinstalled, cleared cache, everything i could think i can do, but all i can see is the card, i see the devices, i can play music but something feels like it's incomplete because i cannot see the card for grouping, individual volume control and nothing in the favorites area.
Can anyone help me with something to make this work? Maybe i'm doing something wrong.

This is my dashboard
![image](https://github.com/johanfrick/custom-sonos-card/assets/134103405/7afea055-6c09-4fc7-8723-1867dba333a6)

This is what i'm seeing when i try to add the card
![image](https://github.com/johanfrick/custom-sonos-card/assets/134103405/df76cc27-9168-468f-bf69-0f9d04bd4316)

**Comment by punxaphil** [comment]
Just press these buttons to go to favorites, grouping and volumes
<img width="231" alt="image" src="https://github.com/johanfrick/custom-sonos-card/assets/26582898/846c0d54-4b87-4f81-a000-7a57feccebac">

**Comment by phateks** [comment]
in other releases there were separate cards for that, in this build are they gone?

**Comment by punxaphil** [comment]
That was changed more than 5 months ago with [v6](https://github.com/johanfrick/custom-sonos-card/releases/tag/v6)
How to use separate sections: https://github.com/johanfrick/custom-sonos-card/blob/master/README.md#using-individual-section-cards

**Reply by phateks** [reply]
ook, thanks, and sorry for buggin'
then it works like it should.

keep up the great work. have fun!!

**Reply by punxaphil** [reply]
Thanks!
---
PunxaPhil
[![image](https://github.com/johanfrick/custom-sonos-card/assets/26582898/9bebc1f6-6bda-4cbd-b596-50d3d2f712de)](https://www.buymeacoffee.com/punxaphil)
